src/StatusQ/Controls/StatusSwitchTab.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+TabButton {
+    id: control
+    contentItem: Item {
+        height: 36
+        MouseArea {
+            id: sensor
+            hoverEnabled: true
+            anchors.fill: parent
+            cursorShape: control.hovered ? Qt.PointingHandCursor : Qt.ArrowCursor
+            onClicked: control.checked = true
```

Comment:
why? if you set `checkable: true` then `checked` changed automatically on press

Reply:
> why? 
Why what exactly? 
I'm setting `control.checked` when the mouseArea emits `clicked` so that the control gets checked. The reason I've done this is because the mouse area shadows the clickable area of the tab itself, making it uncheckable otherwise.

You're saying setting `MouseArea `checkable: true` will make this work as well?

Reply:
Tried it, there's no `checkable` on `MouseArea`